src/applications/vaos/new-appointment/components/ProviderSelectPage/index.jsx

```diff
@@ -25,11 +27,23 @@ export default function SelectProviderPage() {
     patientProviderRelationships,
   } = useGetPatientRelationships();
 
+  // page header setup
   const pageTitle = useSelector(state => getPageTitle(state, pageKey));
+  const singleProviderTitle = `Your ${typeOfCare.name.toLowerCase()} provider`;
```

Comment:
Nice work! Was going to write up another ticket for the header, but no need to now.